test/vsp/testBsm/runTest.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 source /opt/ros/foxy/setup.bash
-source ../../../other/install/setup.bash
+#source ../../../other/install/setup.bash
+source /home/yilin/SIP_project/carma-msgs/install/setup.bash 
+source ../../../mmitss/src/common/MsgTransceiver/CarmaDecoderRos2/install/setup.bash
 
-../../../mmitss/src/common/MsgTransceiver/CarmaDecoderRos2/src/M_CarmaMsgDecoder & sleep 2;pytest test_hostBsm.py
\ No newline at end of file
+../../../mmitss/src/common/MsgTransceiver/CarmaDecoderRos2/src/M_CarmaMsgDecoder & sleep 2;python3 -m test_hostBsm.py
```

Comment:
There is a way to run pytest similar to this. Apparently you can run something like
```bash
python3 -m unittest test_hostBsm.py
```
(https://docs.python.org/3/library/unittest.html)
Have you tried this?

Reply:
Yes I have tries this. I think it is better to keep the original script in order to close this story. Then I can work on debuging the ros2 issue.